crates/wash-cli/src/app/mod.rs

```diff
@@ -328,3 +323,68 @@ fn show_model_history(results: VersionResponse) -> CommandOutput {
     map.insert("revisions".to_string(), json!(results));
     CommandOutput::new(output::list_revisions_table(results.versions), map)
 }
+
+#[cfg(test)]
+mod test {
+    use super::get_lattice_prefix_and_nats_client_from_cmd_opts;
+    use anyhow::Result;
+    use std::env;
+    use wash_lib::{
+        cli::CliConnectionOpts,
+        config::{DEFAULT_CTX_DIR_NAME, DEFAULT_LATTICE_PREFIX, WASH_DIR},
+        context::{fs::ContextDir, ContextManager, WashContext},
+    };
+
+    #[tokio::test]
+    async fn test_lattice_prefix_and_nats_client_from_cmd_opts() -> Result<()> {
```

Comment:
This is a great test to have. Since it's not specific to the `app` commands, would you mind moving it to `wash-lib/src/cli/mod.rs`, where we define `impl TryFrom<CliConnectionOpts> for WashConnectionOptions` ?